pkg/controller/certificatemanager/certificatemanager.go

```diff
@@ -261,7 +281,7 @@ func (cm *certificateManager) getKeyPair(cli client.Client, secretName, secretNa
 		if timeInvalid {
 			return nil, nil, fmt.Errorf("secret %s/%s is not valid at this date", secretNamespace, secretName)
 		}
-		return nil, nil, fmt.Errorf("secret %s/%s must specify ext key usages: %+v", secretNamespace, secretName, requiredKeyUsages)
+		return nil, nil, &CertUsageError{fmt.Sprintf("secret %s/%s must specify ext key usages: %+v", secretNamespace, secretName, requiredKeyUsages)}
```

Comment:
It will print:

> Failed to get certificate: secret tigera-operator/tigera-compliance-server-tls must specify ext key usages: [2 1]

Which are the enum indices, not the value. I'm ok to defer though.